.changeset/sharp-wombats-speak.md

```diff
@@ -0,0 +1,28 @@
+---
+'@backstage/plugin-auth-backend': minor
+---
+
+Allow additional user-defined scopes for Okta auth in config yaml
```

Comment:
This changeset is unnecessarily large. Changesets should inform the user about the high level impact on them when they quickly browse through the changelog after a release. For the end user, all of the defails about what changed internally is of no interest. And documentation-like text is usually wasteful since it bloats the changeset and then just "vanishes" (it's not easily findable at a later time when it's actually needed by the user).

I'd suggest changing this entire file to be just something like

```
Added an optional `additionalScopes` configuration parameter to `okta` providers, that lets you add additional scopes on top of the default ones.
```

Then if you want to explain the parameter more, you can do so in a doc comment on the field itself in `config.d.ts`. For example mentioning that it's space separated. But don't state what the default set of scopes is, since that might change and then your comment is out of date again.